tooltester/Dockstore-2.json

```diff
@@ -0,0 +1,3 @@
+{
+  "fastqReadCounts.countFastqReads.inputFastq": ["s3://human-pangenomics/working/HPRC_PLUS/HG005/raw_data/Illumina/child/5A1-24481579/5A1_S5_L001_R1_001.fastq.gz"]
```

Comment:
Are you using this file? I don't see it referenced anywhere.

Reply:
My bad, it has now been deleted.